app/js/lib/intent-parser/parsers/users-parser.js

```diff
@@ -3,7 +3,13 @@ const PATTERNS = {
     patterns: [
       /^Remind (.+?) (?:to|at|on|by|that|about)\b/i,
       /^Where (?:am|are|is) (.+?) (?:at|on|by)\b/i,
+      /^Where (?:am|are|is) (.+?) supposed to\b/i,
```

Comment:
Let's stick to American English for now. The integration of local dialects and additional languages will have to be taken care of by proper l18n and i18n systems when we implement them.